test/compute_bands.jl

```diff
@@ -7,100 +7,88 @@ if mpi_nprocs() == 1  # not easy to distribute
-        [0.227272727273, 0.227272727273, 0.227272727273],
-        [0.272727272727, 0.272727272727, 0.272727272727],
-        [0.318181818182, 0.318181818182, 0.318181818182],
-        [0.363636363636, 0.363636363636, 0.363636363636],
-        [0.409090909091, 0.409090909091, 0.409090909091],
-        [0.454545454545, 0.454545454545, 0.454545454545],
-        [0.500000000000, 0.500000000000, 0.500000000000],
-        [0.500000000000, 0.500000000000, 0.500000000000],
-        [0.515625000000, 0.468750000000, 0.515625000000],
-        [0.531250000000, 0.437500000000, 0.531250000000],
-        [0.546875000000, 0.406250000000, 0.546875000000],
-        [0.562500000000, 0.375000000000, 0.562500000000],
-        [0.578125000000, 0.343750000000, 0.578125000000],
-        [0.593750000000, 0.312500000000, 0.593750000000],
-        [0.609375000000, 0.281250000000, 0.609375000000],
-        [0.625000000000, 0.250000000000, 0.625000000000],
-        [0.625000000000, 0.250000000000, 0.625000000000],
-        [0.600000000000, 0.250000000000, 0.650000000000],
-        [0.575000000000, 0.250000000000, 0.675000000000],
-        [0.550000000000, 0.250000000000, 0.700000000000],
-        [0.525000000000, 0.250000000000, 0.725000000000],
-        [0.500000000000, 0.250000000000, 0.750000000000],
-        [0.500000000000, 0.250000000000, 0.750000000000],
-        [0.500000000000, 0.277777777778, 0.722222222222],
-        [0.500000000000, 0.305555555556, 0.694444444444],
-        [0.500000000000, 0.333333333333, 0.666666666667],
-        [0.500000000000, 0.361111111111, 0.638888888889],
-        [0.500000000000, 0.388888888889, 0.611111111111],
-        [0.500000000000, 0.416666666667, 0.583333333333],
-        [0.500000000000, 0.444444444444, 0.555555555556],
-        [0.500000000000, 0.472222222222, 0.527777777778],
-        [0.500000000000, 0.500000000000, 0.500000000000],
-        [0.500000000000, 0.500000000000, 0.500000000000],
-        [0.484375000000, 0.484375000000, 0.531250000000],
-        [0.468750000000, 0.468750000000, 0.562500000000],
-        [0.453125000000, 0.453125000000, 0.593750000000],
-        [0.437500000000, 0.437500000000, 0.625000000000],
-        [0.421875000000, 0.421875000000, 0.656250000000],
-        [0.406250000000, 0.406250000000, 0.687500000000],
-        [0.390625000000, 0.390625000000, 0.718750000000],
-        [0.375000000000, 0.375000000000, 0.750000000000],
-        [0.625000000000, 0.250000000000, 0.625000000000],
-        [0.600000000000, 0.200000000000, 0.600000000000],
-        [0.575000000000, 0.150000000000, 0.575000000000],
-        [0.550000000000, 0.100000000000, 0.550000000000],
-        [0.525000000000, 0.050000000000, 0.525000000000],
-        [0.500000000000, 0.000000000000, 0.500000000000],
+        [0.025000000000, 0.000000000000, 0.025000000000],
+        [0.050000000000, 0.000000000000, 0.050000000000],
+	[0.075000000000, 0.000000000000, 0.075000000000],
```

Comment:
I am dubious about this. If everything is done right you should get exactly the same k-points, no?